imports/blocks/give/fieldsets/shared/Icon.js

```diff
@@ -0,0 +1,12 @@
+import { PropTypes } from "react";
+import AccountType from "../../../../components/accountType";
+
+const Icon = ({ cardType }) => (
```

Comment:
just curious, since this just returns `<AccountType>`, why is the extra component needed?